test/IntegrationTests/SmokeTests.cs

```diff
@@ -327,21 +326,22 @@ public void PrometheusAndOtlpMetricsExporter()
         {
             var assert = async () =>
             {
-                var httpClient = new HttpClient
+                for (var i = 0; i < 50; i++)
```

Comment:
maybe should convert the `NotThrowAfterAsync` into internal helper, not to mix asserting logic and something to be asserted.

Reply:
88c41cfe6e3b5f2d1496eb32f756a9491bf916e1, please double check :)